static/js/chart/draw.ts

```diff
@@ -26,15 +26,18 @@ const LEGEND = {
   minTextWidth: 100,
   dashWidth: 30,
   lineMargin: 10,
-  marginLeft: 0,
+  marginLeft: -15,
```

Comment:
A negative margin seems odd.. perhaps the calculation needs to be updated?

Reply:
updated